Supporting Documents/BS_SD.adoc

```diff
@@ -326,6 +326,8 @@ Following input is required from the developer.
 
 [loweralpha]
 . TSS shall explain how the TOE meets FIA_MBE_EXT.2 at high level description
+** If an independent quality metric is claimed, the TSS must include information about the metric
+** If no independent quality metric is claimed, the TSS must include a desription of how quality is measured
```

Comment:
* I think these two bullets say the same thing
 * independent -> internal

```suggestion
** If a non-standard (i.e., internal) quality metric is claimed, the TSS must include information about the metric
```

Reply:
Is it sufficient to only point out expectations if an internal metric is used? Is it clear that an independent standard should also be noted without stating it here?

Reply:
I may have misunderstood a reporting requirement:
 * If a documented standard quality metric is used, I think saying the standard name + threshold (lines 406, 433) is sufficient. The evaluator should be able to refer to the standard for more information.
 * If a proprietary quality metric is used, information about the proprietary metric should be required so the evaluator can attempt to make samples of different quality.

Reply:
Actually I think these should just be removed and different text added later in the description (looking at it). I don't think anything is needed here, or if anything is, it should just state that the quality metric description should be included, and leave it vague since this is actually like an overview section. It needs to be included later in some more detail (not much more, but under the assessment strategies section)